woocommerce/woocommerce
# WCPay Experiment: Create an Inbox Note if user wants more info on WCPay

### All Submissions:






### Changes proposed in this Pull Request:



Closes [automattic/woocommerce-payments/4536](https://github.com/Automattic/woocommerce-payments/issues/4536)






### How to test the changes in this Pull Request:



1. Temporarily comment out the [checks](https://github.com/woocommerce/woocommerce/blob/6b80bbd1aca1e3031bd1f8693eb00b3ca4e707cf/plugins/woocommerce/src/Internal/Admin/WcPayWelcomePage.php#L31-L80) to ensure that the WCPay Promo is always shown. You should see `Payments (1)` In the menu on WP Admin. Note that if WCPayments is already installed you may see two menu items, this wouldn't happen in reality because of the checks we are disabling to test this.

<img width="821" alt="Screenshot 2022-11-14 at 17 53 51" src="https://user-images.githubusercontent.com/11770181/201731493-fe1de9ff-0cf9-4248-8847-419c4d54b4c5.png">

2. Click on this to open the WCPay Promo (see screenshot below). Click `No thanks`, then select `I need more information about WooCommerce Payments`, then click `Dismiss and send feedback`.

<img width="1343" alt="Screenshot 2022-11-14 at 17 54 40" src="https://user-images.githubusercontent.com/11770181/201731675-1fd42dd1-6cb0-4577-94b3-6ea7214b75c0.png">

4. If you have WCPay installed as a plugin, go to `plugins/woocommerce/src/Internal/Admin/Notes/PaymentsMoreInfoNeeded.php` and disable the [check](https://github.com/woocommerce/woocommerce/blob/accd4394af1ec1eba032167fce74cc84f0597649/plugins/woocommerce/src/Internal/Admin/Notes/PaymentsMoreInfoNeeded.php#L43) to see if WCPay is installed. (or temporarily remove WCPay from your store).
5. Go to the database, search the `wp_options` table for the `wc_calypso_bridge_payments_dismissed` option value, and ensure the value is `yes`.
6. Check the `wp_options` table for the `wc_pay_exit_survey_more_info_needed_timestamp` value, and change the value so that it is a timestamp which is more than 30 days in the past (you can use `1636912686`).
7. Install a plugin like `WP Crontrol`, and force a manual run of the `wc_admin_daily` event.
8. Go to `WooCommerce > Home` and verify that you see the new Inbox Note. 
<img width="459" alt="Screenshot 2022-11-15 at 11 37 04" src="https://user-images.githubusercontent.com/11770181/201918663-5c0b6431-2560-437a-ab1c-651fa0869f3d.png">




### Other information:




### FOR PR REVIEWER ONLY:
